server/src/main/java/org/elasticsearch/index/mapper/TextFieldMapper.java

```diff
@@ -339,7 +344,6 @@ public Query existsQuery(QueryShardContext context) {
         PrefixFieldType(String name, int minChars, int maxChars) {
             setTokenized(true);
             setOmitNorms(true);
-            setIndexOptions(IndexOptions.DOCS);
```

Comment:
I think we still need this?  What happens with a 6.3 index that doesn't store offsets?